scripts/e2e/cmd/runner/helper.go

```diff
@@ -397,3 +434,45 @@ func makeGetRequest(url string, host string, statusCode int, inSecure bool) (*ht
 
 	return nil, fmt.Errorf("Unable to get status code %d with url '%s' with host '%s'. Response: [%+v]", statusCode, url, host, resp)
 }
+
+func validateSubResourcePrefix() error {
```

Comment:
I think the resourcePrefix is already covered in your previous PR already? better to do in different PR

Reply:
I forgot to mention that this PR is based on the previous PR, so will remove changes